crates/crypto/src/key/private_key.rs

```diff
@@ -1,9 +1,10 @@
 use crate::key::public_key::PublicKey;
 use crate::key::{Key, KeyError};
+use serde::{Deserialize, Serialize};
 use ssi_jwk::JWK;
 use ssi_jws::sign_bytes;
 
-#[derive(Clone, PartialEq, Debug)]
+#[derive(Clone, PartialEq, Debug, Deserialize, Serialize)]
```

Comment:
don't _love_ this, perhaps there's a more secure way to do this but it's fine for now